cfg.js

```diff
@@ -12,4 +12,5 @@ module.exports = {
 	maxChannelsEarningFrom: 25,
 	limitedIdentityEarningsLimit: '100000000000000000000', // 100 DAI/TST
 	limitedIdentityEnabled: false,
+	clustered: true,
```

Comment:
this should be an ENV var rather than a config (use process.env.CLUSTERED; default to false)